eras/mary/impl/cardano-ledger-mary.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-ledger-mary
-version:            1.4.0.0
+version:            1.5.0.0
```

Only bounds changed:
```suggestion
version:            1.4.0.1
```